examples/using-gatsby-with-json-yaml/src/pages/jsononclient.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import uuid from "uuid"
+import JSONData from "../../content/client-data.json"
+const ClientJSON = () => (
```

Comment:
This is happening at build time, so it shouldn't have the word "Client" in it. The same goes for YAML.